specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/examples/jobs-update.json

```diff
@@ -9,24 +9,17 @@
     "parameters": {
       "properties": {
         "description": "Example job to illustrate update.",
-        "priority": "High",
         "input": {
-          "@odata.type": "#Microsoft.Media.JobInputs",
-          "inputs": [
-            {
-              "@odata.type": "#Microsoft.Media.JobInputAsset",
-              "files": [],
-              "assetName": "job1 input"
-            }
-          ]
+          "@odata.type": "#Microsoft.Media.JobInputAsset",
+          "assetName": "job1-InputAsset"
         },
         "outputs": [
           {
             "@odata.type": "#Microsoft.Media.JobOutputAsset",
-            "label": "example-custom-label",
-            "assetName": "job1 output"
+            "assetName": "job1-OutputAsset"
           }
-        ]
+        ],
+        "priority": "High"
```

Comment:
Did Priority move down here for a reason? Just curious why it moved from below description.

Reply:
these are auto generated tests.. not sure what changed in the generation process.